app/src/main/java/com/sci/coffeeandroid/feature/menudetails/ui/screen/MenuDetailsActivity.kt

```diff
@@ -30,18 +51,36 @@ class MenuDetailsActivity() : AppCompatActivity() {
             insets
         }
 
-        coffeeDetailViewModel.fetchCoffeeDetail(id = 1)
-
-        coffeeDetailViewModel.coffeeDetailLiveData.observe(this){
+        menuDetailsViewModel.coffeeDetailsLiveData.observe(this) {
```

Comment:
if LiveData emit state or event , it should naming should be represent its state or event 
eg coffeeDetailsUiState or coffeeDetailsUiEvent